nsdb-http/src/main/scala/io/radicalbit/nsdb/web/routes/CommandApi.scala

```diff
@@ -182,19 +237,17 @@ trait CommandApi {
       new ApiResponse(code = 500, message = "Internal server error")
     ))
   def showMetrics: Route =
-    pathPrefix("commands") {
-      pathPrefix(Segment) { db =>
-        pathPrefix(Segment) { namespace =>
-          path("metrics") {
-            pathEnd {
-              get {
-                withNamespaceAuthorization(db, namespace, false, authorizationProvider) {
-                  onComplete(readCoordinator ? GetMetrics(db, namespace)) {
-                    case Success(MetricsGot(_, _, metrics)) =>
-                      complete(HttpEntity(ContentTypes.`application/json`, write(ShowMetricsResponse(metrics))))
-                    case Success(_)  => complete(HttpResponse(InternalServerError, entity = "Unknown reason"))
-                    case Failure(ex) => complete(HttpResponse(InternalServerError, entity = ex.getMessage))
-                  }
+    pathPrefix(Segment) { db =>
+      pathPrefix(Segment) { namespace =>
+        path("metrics") {
+          pathEnd {
+            get {
+              withNamespaceAuthorization(db, namespace, false, authorizationProvider) {
+                onComplete(readCoordinator ? GetMetrics(db, namespace)) {
+                  case Success(MetricsGot(_, _, metrics)) =>
+                    complete(HttpEntity(ContentTypes.`application/json`, write(ShowMetricsResponse(metrics))))
+                  case Success(_)  => complete(HttpResponse(InternalServerError, entity = "Unknown reason"))
```

Comment:
does it make sense to you here to log in "entity" field also the object message type you have as the return of this success? Maybe helps debugging session.